packages/coinstac-simulator/test/mocks/good-declaration-1.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  computationPath: './path/to/computation.js',
+  local: Array(10),
```

Comment:
Little addition to the API to make client instantiation mostly effortless. We could also change this to simply accept the number:

```js
module.exports = {
  computationPath: './path/to/computation.js',
  local: 10,
};
```